requirements.txt

```diff
@@ -0,0 +1,3 @@
+google-cloud-core==1.6.0
+google-cloud-documentai==0.3.0
+google-cloud-storage==1.36.2
```

Comment:
I tested out the bounding boxes in the generic form parser a couple times, I *believe* we don't need any additional packages here

Reply:
also will add the bounding boxes back to the specialize parser in a follow up PR